### ISSUE TITLE
misnaming of {x,y,z}_grid in neutrals/grid? 

### ISSUE THREAD

**Issue by shaunhaskey** [D3DEnergetic/FIDASIM#83]
@lstagner @bgrierson21 I was trying to get all of my old FIDASIM plotting routines to work and track down the differences in the intensities when I noticed that in the neutrals.h5 (and dcx) file the values in grid/x_grid appear to be in machine coordinates (so shouldn't it be named u_grid instead?). Same with y_grid, and z_grid. On the other hand the values in the 1D array grid/x are in FIDASIM co-ordinates, so they seem to be correctly named.

**Comment by lstagner** [comment]
It was actually intentional. In the last version there was confusion with the xyz-uvw naming convention. I was thought that since xyz is more common it should represent the machine coordinates and uvw the beam grid coordinates. But in the original FIDASIM paper xyz was used for the beam grid and uvw for the machine. The fact that the uvw and xyz could be the same was also a source of confusion.

In the new version I decided that in the facing code(I still use uvw-xyz in fidasim) I would used xyz for everything and use the description attribute to indicate what the coordinate system was. I figured if users accidentally used the wrong coordinate system it would be obvious.

I could, of course, revert the changes if that is what most people want.
